src/app/form-builder/edit-panel/edit-panel.component.ts

```diff
@@ -226,7 +226,14 @@ export class EditPanelComponent implements OnInit, AfterViewInit {
         this.gridsterService.options.minCols = newCols;
         this.gridsterService.options.maxCols = newCols;
         this.gridsterService.options.maxItemCols = newCols;
-        this.modelService.model.getTransition(this.transId).dataGroups[0].cols = newCols;
+        if (this.modelService.model.getTransition(this.transId).dataGroups?.length > 0) {
+            this.modelService.model.getTransition(this.transId).dataGroups[0].cols = newCols;
+        } else {
+            const dataGroup = new DataGroup(`${this.transId}_0`);
```

Comment:
Creating new DataGroup would be better when opening the form builder. This way you will guarantee that a data group exist not only for this component but also for others that might need it.

Reply:
done